content/en/pages/home.jade

```diff
@@ -5,8 +5,8 @@ block homepage-header
 		img(src='/images/logo-inverted.svg', alt='KeystoneJS', width=80, height=81)
 		h1 Node.js CMS &amp; Web Application Platform
 		h2 The open source framework for developing database-driven websites, applications and APIs in Node.js. Built on Express and MongoDB.
-		.home-header-buttons
-			a(href='/getting-started', title='KeystoneJS getting started guide').btn.btn-lg.btn-primary
+		.home-header-buttons 
+			a(href='/quick-start', title='KeystoneJS quick start guide').btn.btn-lg.btn-primary
```

Comment:
:+1:

Once this is merged maybe @weekwood can help with the rest of the translation